File: Lab_C++/biblioteca.cpp
#include <iostream>
using namespace std;

int main() 
{
    string estadoTarjeta;
    // Solicitar al usuario el estado de la tarjeta de la biblioteca
    cout << "La tarjeta de la biblioteca esta activa (Si/No): ";
    cin >> estadoTarjeta;

    // Verificar si la tarjeta está activa
    if (estadoTarjeta == "Si" || estadoTarjeta == "si" || estadoTarjeta == "SI") {
        // Si la tarjeta está activa, el usuario puede realizar préstamos
        cout << "Puedes realizar prestamos." << endl;
    } else {
        // Si la tarjeta no está activa, no se permiten préstamos
        cout << "No se permiten prestamos." << endl;
    }

    return 0;
}

File: Lab_Java/HelloWorld/src/Biblioteca.java
import java.util.Scanner;

public class Biblioteca {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        // Esto solicita al usuario qu ingrese si esta activa o no
        System.out.print("¿La tarjeta de la biblioteca esta activa? (Si/No)");
        String estadoTarjeta = scanner.nextLine();

        // Esto verifica si la tarjeta esta activa
        if (estadoTarjeta.equalsIgnoreCase("Si")) {
            // Si esta activada puede usar prestamos
            System.out.println("Puedes realizar prestamos.");
        }

        scanner.close();
    }
}
